### DISCUSSION TITLE
Export tags and collections to Obsidian

### DISCUSSION THREAD

**Discussion by labosca** [retorquere/zotero-better-bibtex#2524] [Q&A]
I'm having issues setting up my template for Zotero Notes import into Obsidian. It fails exporting tags and collection names, leaving the fields empty, when using Zotero Integration plugin. (if I export a .md file directly from Zotero with Mdnotes, then it's correctly compiled). Could it be a problem related to BetterBibTex? Do anybody have a possible solution to fix it?

**Comment by retorquere** [comment]
BBT doesn't do anything werkt templates, so I think it's unlikely it's involved in the workflow you describe.

**Reply by labosca** [reply]
Thank you for your reply. I thought that maybe, since Zotero Integration plugin installed in Obsidian retrieves metadata using BBT "instructions", I should have to fix the way BBT translates informations. In the templates I'm just telling, for example "keywords:: {{tags}}", but it won't recognize any tag. I don't know if I made myself clear. Otherwise, do you have any suggestion for what might be the problem?

**Reply by retorquere** [reply]
I have no idea how the Obsidian plugin works or what data it does or does not retrieve, sorry. If the developers of that plugin can tell me what they are calling in BBT and what information is missing, I can join that conversation, but it will primarily be a conversation between them and you.

**Reply by labosca** [reply]
No worries, thank you for your support. I managed thanks to this discussion https://github.com/mgmeyers/obsidian-zotero-integration/issues/117 so I looked to Data Explorer in Obsidian and fixed accordingly to parameters name my template requests. 
Thank you again
